distribution/docs/src/main/resources/content/_metadataAttributes/NITF_Segment_Image-table.adoc

```diff
@@ -117,6 +117,16 @@
 |String
 |true
 
+|ImageSegment.TGTID
+|The last 2 characters of ImageSegment.TGTID correspond to the country code. If these characters are not spaces then null. Else,
+
+2char FIPS -> 3char ISO3
+
+If conversion fails, then null. NOTE: Other system plugins could populate the country code based on location.
+|<<location.country-code,location.country-code>>
+|String
+|true
```

Comment:
![image](https://user-images.githubusercontent.com/8041246/43423487-6991c5f6-9401-11e8-8cb8-657a504fa2f8.png)
I confirmed that these hyperlinks jump to the right place.

Reply:
I just realized that the` ext.nitf.image.target-identifier` row needs to be updated to not say trimmed.